libs/financial-aid/shared/src/lib/taxCalculator.ts

```diff
@@ -13,11 +13,14 @@ export interface TaxInfo {
   personalTaxAllowance: number
   taxPercentage: number
 }
+import format from 'date-fns/format'
+import { Amount, Calculations } from './interfaces'
+
+export const currentYear = format(new Date(), 'yyyy')
```

Comment:
Is this imported in other places?

Reply:
We weere always sending it as props, but nor really using it again, so I thought this would be better?

Reply:
Just wondering if it should be exported? And if it should be exported then it should be in another file :)